### PR TITLE
Fix PHP notice with success_action

### PR DESCRIPTION
**Pull request by truongwp**
Fixes https://github.com/Strategy11/formidable-pro/issues/4068

### CODE REVIEW

**Review by stephywells — APPROVED** [review]
Thank you @truongwp !

File: classes/controllers/FrmFormsController.php [hunk 1]
```diff
@@ -2929,10 +2929,10 @@ private static function maybe_migrate_submit_settings_to_action( $form_id ) {
 	private static function get_on_submit_action_data_from_form_options( $form_options, $event = 'create' ) {
 		$opt  = 'update' === $event ? 'edit_' : 'success_';
 		$data = array(
-			'success_action' => $form_options[ $opt . 'action' ],
+			'success_action' => isset( $form_options[ $opt . 'action' ] ) ? $form_options[ $opt . 'action' ] : FrmOnSubmitHelper::get_default_action_type(),
 		);
 
-		switch ( $form_options[ $opt . 'action' ] ) {
+		switch ( $data['success_action'] ) {
```
Comment (stephywells) [inline classes/controllers/FrmFormsController.php:2935]: Actually.. This line isn't quite right. This lost the $opt

**Review by stephywells — CHANGES_REQUESTED** [review]

File: classes/controllers/FrmFormsController.php (hunk 1 above)
Comment (truongwp) [reply classes/controllers/FrmFormsController.php:2935]: I use `$data`, not `$form_options`.

Comment (stephywells) [reply classes/controllers/FrmFormsController.php:2935]: What happens if the event is 'edit' instead of 'success'?

Comment (truongwp) [reply classes/controllers/FrmFormsController.php:2935]: In the On Submit action, we just use `success_action`, not matter it's an `update` or `create` event.

Comment (truongwp) [reply classes/controllers/FrmFormsController.php:2935]: We just need to check in the `$form_options`: https://github.com/Strategy11/formidable-forms/pull/1065/files#diff-1b880a2d11e84622570383ea0503da990c04c1ccbffe4f5f81ab21d68fa00280R2932

Comment (stephywells) [reply classes/controllers/FrmFormsController.php:2935]: Oh! I see it now. It's all good.

**Review by stephywells — APPROVED** [review]

**Pull request merged**